wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/downloads/DownloadsScreen.kt

```diff
@@ -0,0 +1,200 @@
+}
+
+@Composable
+private fun Content(
+    columnState: ScalingLazyColumnState,
+    episodes: List<Episode>,
+    onItemClick: (Episode) -> Unit,
+) {
+    ScalingLazyColumn(
+        columnState = columnState,
+    ) {
+        item {
+            Text(
+                text = stringResource(
+                    if (episodes.isEmpty()) {
+                        LR.string.profile_empty_downloaded
+                    } else {
+                        LR.string.downloads
+                    }
+                ),
+                textAlign = TextAlign.Center,
+                style = MaterialTheme.typography.button,
+                modifier = Modifier
+                    .padding(horizontal = 10.dp)
+                    .padding(bottom = 12.dp)
+                    .fillMaxWidth(),
+            )
+        }
+
+        items(episodes) { episode ->
+            Download(
+                episode = episode,
+                onClick = { onItemClick(episode) }
+            )
+        }
+    }
+}
+
+@Composable
+private fun Download(episode: Episode, onClick: () -> Unit) {
+    Box(
+        contentAlignment = Alignment.Center,
+        modifier = Modifier
+            .clip(RoundedCornerShape(24.dp))
+            .background(MaterialTheme.colors.surface)
+            .clickable { onClick() }
+            .padding(horizontal = 10.dp)
+            .fillMaxWidth()
+            .height(72.dp)
+    ) {
+        Row {
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                PodcastImage(
+                    uuid = episode.podcastUuid,
+                    dropShadow = false,
+                    modifier = Modifier.size(30.dp),
+                )
+                Spacer(Modifier.height(4.dp))
+                Icon(
+                    painter = painterResource(IR.drawable.ic_downloaded),
+                    contentDescription = null,
+                    tint = MaterialTheme.theme.colors.support02,
+                    modifier = Modifier.size(12.dp),
+                )
+            }
+
+            Spacer(Modifier.width(6.dp))
+
+            Column(
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                Text(
+                    text = episode.title,
+                    lineHeight = 14.sp,
+                    overflow = TextOverflow.Ellipsis,
+                    style = MaterialTheme.typography.button.merge(
+                        @Suppress("DEPRECATION")
+                        TextStyle(
+                            platformStyle = PlatformTextStyle(
+                                // So we can align the top of the text as closely as possible to the image
+                                includeFontPadding = false,
+                            ),
+                        )
+                    ),
+                    maxLines = 2,
+                )
+                val shortDate = episode.publishedDate.toLocalizedFormatPattern("dd MMM")
```

Comment:
Nitpick: I see that if the duration is less than an hour, mins are displayed slightly differently:

<img width="185" alt="Screenshot 2023-03-31 at 9 08 26 AM" src="https://user-images.githubusercontent.com/1405144/229016790-6274f95d-48a8-434f-b124-3a679505dfe1.png">

No need to fix it right now, just mentioning it as I noticed it in Figma.

Reply:
Thanks. I'll confirm with Adam.